src/plugins/paginate/paginate-en.hbs

```diff
@@ -0,0 +1,839 @@
+		<li class="wb-pgfltr-out">Veritatis perspiciatis eligendi earum assumenda deleniti quibusdam omnis cumque quisquam!</li>
+		<li class="wb-pgfltr-out">Libero molestias explicabo natus id aliquam veniam suscipit? Repudiandae, minima!</li>
+		<li class="wb-pgfltr-out">Labore nesciunt numquam modi minus nostrum eveniet temporibus expedita ea.</li>
+		<li class="wb-pgfltr-out">Neque, eveniet deserunt. Quasi suscipit nulla similique porro perferendis atque.</li>
+		<li class="wb-pgfltr-out">Quo laborum quos eos quas vitae eveniet expedita cupiditate incidunt?</li>
+		<li class="wb-pgfltr-out">Sit, at doloribus deserunt magnam quasi perferendis reprehenderit reiciendis eligendi?</li>
+		<li class="wb-pgfltr-out">Fugiat eos modi voluptatum sit amet voluptas fugit necessitatibus veniam!</li>
+		<li class="wb-pgfltr-out">Laudantium sunt earum voluptatum cupiditate aut rem tempore nam ea.</li>
+		<li class="wb-pgfltr-out">Odit expedita sunt distinctio reiciendis dolores magnam nobis non. Reiciendis.</li>
+		<li class="wb-pgfltr-out">Corrupti suscipit, sunt eligendi exercitationem distinctio consequatur repudiandae numquam dicta.</li>
+		<li class="wb-pgfltr-out">Dolore, saepe veritatis. Voluptatibus temporibus laboriosam impedit nisi quod perferendis?</li>
+		<li class="wb-pgfltr-out">Nesciunt, reprehenderit rem quis odio incidunt laborum aliquid dolorem nihil.</li>
+		<li class="wb-pgfltr-out">Cum fuga, maxime quod quos sit quidem odit atque rem!</li>
+		<li class="wb-pgfltr-out">Harum sint soluta veniam blanditiis laboriosam corporis repellendus aspernatur unde?</li>
+		<li class="wb-pgfltr-out">Exercitationem dolorem quas quae ab, praesentium obcaecati quis similique culpa!</li>
+		<li class="wb-pgfltr-out">Nesciunt ratione modi sit, placeat quo sint perspiciatis consequatur nam.</li>
+		<li class="wb-pgfltr-out">Saepe cumque, reprehenderit tempore neque dolores quo eum velit ullam?</li>
+		<li class="wb-pgfltr-out">Praesentium neque, quae molestias sed mollitia doloremque temporibus. Omnis, placeat?</li>
+		<li class="wb-pgfltr-out">Quos magni at exercitationem corporis nulla quae pariatur placeat blanditiis?</li>
+		<li class="wb-pgfltr-out">Sit, id? Beatae eveniet fugit ut sapiente accusamus modi nisi.</li>
+		<li class="wb-pgfltr-out">Quam, in tenetur dolore quis aspernatur ipsum adipisci officia perferendis.</li>
+		<li class="wb-pgfltr-out">Dicta sapiente non at, maiores sunt natus asperiores nihil ducimus.</li>
+		<li class="wb-pgfltr-out">Accusamus inventore ad explicabo earum quidem qui soluta iusto officia.</li>
+		<li class="wb-pgfltr-out">At rerum doloremque nostrum perspiciatis repellat velit nihil aperiam laborum!</li>
+	</ul>
+</section>
+
+<section id="paginatedTable">
+	<h2>Example 2 - Table</h2>
+	<p>This example leverages the following option: <code>startPage</code></p>
+	<table class="table wb-paginate provisional" data-wb-paginate='{
+		"startPage": 3
```

Comment:
I will need more info about this use case of when we need to configure statically the start page.

I do have the impression is for when, on page load, an element in deeper page are receiving the focus. In that situation we can probably use the `document.activeElement` to figure out what page to auto-navigate toward.

I can also see it configurable with do-action plugin or the url-mapping plugin for example

Reply:
Removed "startPage" option. The feature to navigate to active element's page on init will be added at a later date.